packages/help-center/src/components/utils.tsx

```diff
@@ -40,14 +43,22 @@ export const generateContactOnClickEvent = (
 	}
 };
 
-export const getLastMessage = ( {
+/**
+ * Returns the last message from a conversation and the number of unread messages.
+ * @returns An object containing the last message and the number of unread messages.
+ */
+export const getLastMessageAndUnread = ( {
 	conversation,
 }: {
 	conversation: OdieConversation | ZendeskConversation;
-} ) => {
-	return Array.isArray( conversation.messages ) && conversation.messages.length > 0
-		? conversation.messages[ conversation.messages.length - 1 ]
-		: null;
+} ): { lastMessage: OdieMessage | ZendeskMessage | null; unreadMessages: number } => {
+	const lastMessage =
+		Array.isArray( conversation.messages ) && conversation.messages.length > 0
+			? conversation.messages[ conversation.messages.length - 1 ]
+			: null;
+
+	const { unreadMessages } = calculateUnread( [ conversation as ZendeskConversation ] );
+	return { lastMessage, unreadMessages };
```

Comment:
We still call `getLastMessageAndUnread()` to retrieve only the last message half of the time, so I don't see much value in grouping the retrieval of the last message with the computation of the number of messages. What about calling them separately?

If that was me, I would even rename `calculateUnread()` into something more explicit such as `getNumberOfUnreadConversationsAndMessages()`. With the latter it's easy to understand what that function does, with the former I would have to look at the code to see what output I should expect.